app/models/image.rb

```diff
@@ -19,4 +20,16 @@ class Image < ActiveRecord::Base
   scoped_search :in => :operatingsystem, :on => :id, :rename => "operatingsystem", :complete_enabled => false, :only_explicit => true
   scoped_search :on => :user_data, :complete_value => {:true => true, :false => false}
 
+  private
+
+  def uuid_exists?
```

Comment:
Please refactor to: 

```ruby
    return true unless compute_resource.present?
    errors.add(:uuid, :invalid) unless compute_resource.image_exists?(uuid)
```

Just so Rubocop won't complain when we enable [this cop](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)